modules/menuItems.js

```diff
@@ -357,9 +358,17 @@ let menuTempl = function (webviews) {
                 log = 'Couldn\'t load log file.';
             }
         },
-    },
-    ];
+    }];
 
+    if (Settings.uiMode === 'mist') {
+        devToolsMenu.push({
+            label: i18n.t('mist.applicationMenu.develop.browsersol'),
```

Comment:
I think it is better to use `remix` throughout instead of `browsersol` or `BrowserSolidity` (below).